packages/@vuepress/theme-default/src/client/styles/vars.scss

```diff
@@ -65,6 +65,17 @@
   --sidebar-width-mobile: calc(var(--sidebar-width) * 0.82);
   --content-width: 740px;
   --homepage-width: 960px;
+
+  // transition vars
+  --t-options: 0.3s ease;
+  --t-p-all: all var(--t-options);
+  --t-p-opacity: opacity var(--t-options);
+  --t-p-transform: transform var(--t-options);
+  --t-p-color: color var(--t-options);
+  --t-p-border-color: border-color var(--t-options);
+  --t-p-background-color: background-color var(--t-options);
+  --t-color-all: var(--t-p-color), var(--t-p-border-color),
+    var(--t-p-background-color);
```

Comment:
Well, both @Shigma and me think we might not need so many vars.

IMO, only the `--t-options` is helpful for reusing the transition options. 🤔

Reply:
Although this PR is intended to provide scalability, I believe consistency is also important to a theme.

For theme-default, two transition variables are enough. One for color (color, border-color, background-color, etc) and the other for position (left, right, top, bottom, width, height, transform, etc).

Reply:
3906192

What naming convention is better to use in `_variables.scss`?

Reply:
I think using `color var(--t-color-options)` directly is enough 🤔  No need to add extra scss variables

Reply:
2908205